pages/establishment/formatters/flag-grouping.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  groupFlags: establishment => {
+    establishment.licences = [];
+    ['supplying', 'breeding', 'procedure'].map(licence => {
+      if (establishment[licence] === true) {
+        establishment.licences.push(licence);
+      }
+    });
+
+    establishment.authorisations = [];
+    ['killing', 'rehomes'].map(authorisation => {
+      if (establishment[authorisation] === true) {
+        establishment.authorisations.push(authorisation);
+      }
+    });
```

Comment:
These `.map.push` arrangements can be much simplified by using `.filter` instead.

```js
establishment.licences =  ['supplying', 'breeding', 'procedure'].filter(licence => establishment[licence]);
establishment.authorisations =  ['killing', 'rehomes'].filter(licence => establishment[licence]);
```